standard/clause_4_terms_and_definitions.adoc

```diff
@@ -1,17 +1,23 @@
 == Terms and Definitions
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in IETF RFC 2119.
 
-For the purposes of this document, the following additional terms and definitions apply.
+For the purposes of this document, the following additional terms and definitions apply:
 
-=== *RFC*
-Request For Comment (IETF)
-
-=== *BCP*
-Best Current Practice. It may include several RFCs (IETF)
+=== *BCP* (Best Current Practice)
+subset of RFC documents designated as best current practice [IETF]
 
 === *Coverage*
-feature that acts as a function to return values from its range for any direct position within its spatial, temporal or spatiotemporal domain (ISO19123-1)
+feature that acts as a function to return values from its range for any direct position within its spatial, temporal or spatiotemporal domain [ISO19123]
+
+=== *Domain*
+well-defined set [ISO19123]
```

Comment:
Seems like a very brief definition, but this is indeed what is in ISO19123! I thought it was something like "set of direct positions for which the coverage function returns a value". Maybe that definition is somewhere else